extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -127,6 +123,8 @@ easy enough to change the loader to ignore `accessor` without `bufferView` in gl
 definitely open to change this if there actually are some use cases that require
 loading `accessor` independently. 
 
+For effectiveness, it is also valid to not support the fallback which is provided in the above specification, e.g. not providing the data buffer for uncompressed assets. To do that, a glTF file could just ignored the `bufferViews` of `accessors` belong to a `primitive`. This is valid because in glTF 2.0, `bufferView` is not required in `accessor`, although if it is not present, it will be used with `sparse` field to act as a sparse accessor. In this case, Draco must be considered a required extension and should report error if a loader doens't support it.
```

Comment:
This paragraph looks much better to me. Thanks Don!
Yes, I was also a bit confused by the concept of `extensionRequired`. It seems to me that if glTF requires forward compatibility, then it probably means every new feature as an extension from now on will not be able to use a `extensionRequired` extension. But anyway.